src/protocol/securemul.rs

```diff
@@ -247,19 +226,19 @@ mod tests {
     use rand::Rng;
     use rand_core::RngCore;
 
-    use crate::replicated_secret_sharing::ReplicatedSecretSharing;
+    use crate::secret_sharing_schemes::replicated_secret_sharing::ReplicatedSecretSharing;
```

Comment:
secret sharing is repeated 3 times here, maybe we can have `secret_sharing` module that exports `Replicated` and `Shamir`?